db/migrate/20170704073821_add_problem_summary_and_suggested_fix_to_link_checker.rb

```diff
@@ -0,0 +1,6 @@
+class AddProblemSummaryAndSuggestedFixToLinkChecker < ActiveRecord::Migration
+  def change
+    add_column :link_checker_api_report_links, :problem_summary, :text
```

Comment:
I wonder if we should make this `null: false` (with a default of `""` to backfill the records), because then we only have one kind of empty string not two (`nil` and `''`).

Reply:
After trying to set these attributes - and getting errors - it turns out we can't set a default on a `BLOB/TEXT` field in MySQL:

>BLOB and TEXT columns cannot have DEFAULT values.

From the [docs](https://dev.mysql.com/doc/refman/5.7/en/blob.html)

Reply:
Oh, that's annoying 😢 Can we back fill the current reports manually with empty strings and then make it `null: false`?

Reply:
We could, Would this not then throw an error when a link comes back as ok from the service?
 As `:problem_summary` or `:suggested_fix` come back as `nil` in the payload.

Reply:
Will set an default in the fetch to avoid this